book/03-git-branching/sections/rebasing.asc

```diff
@@ -2,27 +2,27 @@
 === Rebasing
 
 (((rebasing)))
-In Git, there are two main ways to integrate changes from one branch into another: the `merge` and the `rebase`.
-In this section you'll learn what rebasing is, how to do it, why it's a pretty amazing tool, and in what cases you won't want to use it.
+Pada Git, ada dua cara utama untuk mengintegrasikan perubahan dari satu cabang ke cabang yang lain: `merge` dan `rebase`.
+Pada bagian ini Anda akan belajar apa yang di maksud rebasing, bagaimana melakukannya, mengapa alat ini sangat menakjubkan, dan dalam kasus apa Anda tidak ingin menggunakannya.
 
-==== The Basic Rebase
+==== Dasar Rebase
 
-If you go back to an earlier example from <<_basic_merging>>, you can see that you diverged your work and made commits on two different branches.
+Jika Anda kembali pada contoh sebelumnya <<_basic_merging>>, Anda bisa melihat bahwa Anda telah menyimpang dari pekerjaan Anda dan melakukan commits pada dua cabang yang berbeda.
 
-.Simple divergent history
+.Riwayat yang berbeda
 image::images/basic-rebase-1.png[Simple divergent history.]
 
-The easiest way to integrate the branches, as we've already covered, is the `merge` command.
-It performs a three-way merge between the two latest branch snapshots (`C3` and `C4`) and the most recent common ancestor of the two (`C2`), creating a new snapshot (and commit).
+Cara termudah untuk mengintegrasikan pencabangan, seperti yang sudah kita bahas, adalah perintah `merge`.
+ia akan melakukan penggabungan tiga arah diantara dua snapshot pencabangan terbaru (`C3` dan `C4`) dan nenek moyang yang paling baru dari keduanya (`C2`), membuat snapshot baru (dan commit).
```

Comment:
Typo ia -> Ia.
Untuk "the most recent common ancestor of the two" terjemahkan saja menjadi "commit pendahulu kedua branch tersebut".